dev-python/sphinx-press-theme/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
Add python@ please (or yourself).

We prefer python@ only for stuff with tests though..